homeworks/kirkovvalentiniván/2_vars_datatypes/fictive_character.py

```diff
@@ -0,0 +1,18 @@
+character_name = input("What is your character's name?  ").title().strip()  # Name input
+
+character_age = int(input("How old is your character?  "))                  #Age input
+character_age_days = round(character_age * 365, 0)                          #Age input in days
```

Comment:
A round() itt nem szükséges, mivel a character_age-et int-té alakítod, aztán szintén egy int-tel szorzod meg (365), így az eredmény fixen int lesz, amit nem kell round-olni.

Reply:
Én is fölöslegesnek tartottam, de ha jól emlékszem a házi leírásban szerepelt, hogy kerekítsünk és emiatt beletettem. Itt gondolkoztam is, hogy lehet nem jó a megoldásom, mert nem igényel kerekítést, de javítottam és köszi az észrevételt!